### DISCUSSION TITLE
Great work!

### DISCUSSION THREAD

**Discussion by rdenniston** [runevision/LayerProcGen#3] [General]
This isn't an issue and feel free to close it but I just wanted to say great work from one dev to another - I saw this on /r/proceduralgeneration but I don't have a reddit account (and don't intend to make one), so I figured I would leave the message here. 

Enjoy the weekend!

**Comment by runevision** [comment]
Thanks @rdenniston I appreciate that!

I just added Discussions support to the repo now so there's a better place for non-issue comments and discussion, and I converted your issue to the discussion here.

Have a great weekend too!
